akka-serialization-jackson/src/test/scala/doc/akka/serialization/jackson/SerializationDocSpec.scala

```diff
@@ -119,5 +127,103 @@ object SerializationDocSpec {
     #//#whitelist-class-prefix
   """
 
+  object CustomAdtSerializer {
```

Comment:
I wonder if it'd be easier to put the whole example in its own file?

Reply:
good suggestion, fixed in 94c0586